daemon/cmld.c

```diff
@@ -1190,6 +1205,36 @@ cmld_init(const char *path)
 	else
 		INFO("tss initialized.");
 
+	// Read the provision-status-file to set provisioned flag of control structs accordingly
```

Comment:
why doing this so complex? just use the existence of the file as flag.
There is no need to read and write the content of the file.

just use file_touch() (from common/file.h) in set_provisioned.

Reply:
Done